File: 1-Iniciante/ex1174.c
#include <stdio.h>

int main(){
    /*variaveis*/
    int i;
    float a[100];

    /*entrada*/
    for(i = 0; i < 100; i++){
        scanf("%f", &a[i]);
    }

    /*processamento*/
    for(i = 0; i < 100; i++){
        if(a[i] <= 10){
            printf("A[%d] = %.1f\n", i, a[i]); //saida
        }
    }
    return 0; 
}

File: 1-Iniciante/ex1174.js
// entrada
let input = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n'); // adicione uma barra "/" antes do "dev" para ser aceito no beecrowd
let vetor = input.map(v => parseFloat(v));

// processamento
for(let i = 0; i < vetor.length; i++){
    if(vetor[i] <= 10.0) console.log(`A[${i}] = ${vetor[i].toFixed(1)}`);
}